src/components/events/EventCard.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/Card';
+
+import { Avatar, AvatarImage } from '@/components/ui/Avatar';
+import { Badge } from '@/components/ui/Badge';
+import { cx } from '@/lib/utils';
+import { getTranslations } from 'next-intl/server';
+import Link from 'next/link';
+
+type EventCardProps = {
+  id: number;
+  title: string;
+  subheader: string;
+  description: string;
+  imagePath?: string;
+  startTime: Date;
+  endTime: Date;
+  internal: boolean;
+  locale: string;
+  wrapperClassName?: string;
+  cardClassName?: string;
+  _active?: boolean;
+};
+
+/**
+ * A card for an event.
+ * Only set the _active prop to true if you're testing active events.
+ */
+async function EventCard(props: EventCardProps) {
+  const t = await getTranslations('events');
+
+  const dateOptions = { hour: '2-digit', minute: '2-digit' } as const;
```

Comment:
You don't use hour12: false here, but you do use it in [id]/page.tsx. I would add it here to match with the time formatting in shift schedule page.